2022/tpac/agenda.md

```diff
@@ -0,0 +1,42 @@
+# Privacy CG @ TPAC 2022
+
+## Logistics
+
+The Privacy CG will meet at 13:30-15:30 local Vancouver time
+([America/Vancouver](https://www.timeanddate.com/worldclock/canada/vancouver),
+PDT, UTC-7).
+
+This will be a hybrid in-person and remote meeting for W3C TPAC 2022.  Remote
+access details are available to members through [the TPAC
+calendar](https://www.w3.org/events/meetings/5c485220-8b07-4574-83d8-47a55b50ed1d).
+
+The W3C maintain a [TPAC 2022](https://www.w3.org/2022/09/TPAC/) page with
+details for [registration](https://www.w3.org/register/tpac2022),
+[schedule](https://www.w3.org/calendar/tpac2022/), and other important
+information about participation in TPAC.
+
+Note that remote participation requires registration, which includes a [EUR 240
+fee](https://www.w3.org/2022/09/TPAC/registration.html#fees), though the W3C
+offers a [fee waiver](https://www.w3.org/2022/09/TPAC/registration.html#waiver)
+for those who might find a fee to be a barrier to participation.
+
+[W3C Privacy CG Calendar](https://www.w3.org/groups/cg/privacycg/calendar)
+
+## Agenda
+
+* Setup, Welcome, Code of Conduct (chairs; 10m)
+* FedCM CG Coordination (Heather Flanagan: 30m)
+  * Updates on FedCM work
+  * Discussion of privacy-related topics
+  * Decision tree flows
+* Overview of group, brief status of open work items and proposals (chairs; 20m)
+* Storage Access (Johann ?; 40m)
```

Comment:
Happy to present something, won't be super polished but I can give a quick intro.

Reply:
Thanks.  We're hoping to develop a plan to graduate this work, so focus on those open issues you think need to be more urgently resolved and where you think this work needs to go.